### DISCUSSION TITLE
Configure POS to Print Receipts in A5 Format

### DISCUSSION THREAD

**Discussion by AzibZafar** [microsoft/Dynamics365Commerce.Solutions#341] [Q&A]
Hi Community,

We are currently using the cloud POS with Commerce Scale Unit and have a requirement to print receipts in A5 format.

Could you please advise if it’s possible to achieve this either with or without customization? Additionally, it would be greatly appreciated if you could share any relevant documentation or samples that could assist us in implementing this functionality.

Many thanks in advance!

**Comment by dsdharmen91** [comment]
We can print and design A5 receipt using customization. It can be graphical or any formed invoice with image or graphics. Using RDLC report in POS. you can reach to dsdharmen91@gmail.com

**Reply by cleonmukul** [reply]
Hi dsdharmen91,

We have the same requirements, except that it's required for Store Commerce. I have experience with SSRS, RDLC, and PBI Paginated. I have a lot of questions, such as where the report is hosted? Could you guide me through how you implemented your solution?
